src/acquisition/flusurv/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Column
+from sqlalchemy.ext.declarative import declarative_base
```

Comment:
question: what's the difference between `sqlalchemy.ext.declarative.declarative_base` and `sqlalchemy.orm.declarative_base`?

[The 1.4 documentation](https://docs.sqlalchemy.org/en/14/orm/declarative_tables.html) uses the `.orm` name. Is `.ext` maybe for sqlalchemy 1.3? [Our requirements.txt lists 1.4.40.](https://github.com/cmu-delphi/delphi-epidata/blob/4dc5f129704337a83bbd1ecd3bf2ad573d1e1e5f/requirements.api.txt#L13)

Reply:
basically, they are the same but you are right ist more logical to use sqlalchemy.orm.declarative_base

Reply:
Done